Referenced by issue #41584, bpy.utils.time_from_frame returns wrong values
